remill/BC/Util.cpp

```diff
@@ -1399,11 +1401,11 @@ RecontextualizeType(llvm::Type *type, llvm::LLVMContext &context,
       break;
     }
 
-    case llvm::Type::VectorTyID: {
-      auto arr_type = llvm::dyn_cast<llvm::VectorType>(type);
+    case llvm::getFixedVectorTypeId(): {
```

Comment:
Is this a `constexpr` function? What does it return? Is this the same for old versions of LLVM?

Reply:
Yes and yes :)

I added in a compat stub to define FixedVectorType (as VectorType) for LLVM < 11, and also to correspondingly add a way to get the TypeID.

```
constexpr auto getFixedVectorTypeId() {
#if LLVM_VERSION_NUMBER < LLVM_VERSION(11, 0)
 return Type::VectorTyID;
#else
 return Type::FixedVectorTyID;
#endif
}
```

Reply:
changed to `GetFixedVectorTypeId`